packages/features/ee/teams/lib/removeMember.ts

```diff
@@ -10,10 +12,12 @@ const removeMember = async ({
   memberId,
   teamId,
   isOrg,
+  redirectTo,
 }: {
   memberId: number;
   teamId: number;
   isOrg: boolean;
+  redirectTo?: number;
```

Comment:
`redirectToUserId` is more clear as `redirectTo` could be `username`, team slug etc.